Gemfile

```diff
@@ -51,7 +51,9 @@ group :test do
   # test helpers
   gem "factory_girl_rails",        "~> 4.7"
   gem "fixture_builder",           "0.5.0"
+  gem "json-schema-rspec", "0.0.4"
   gem "rspec-collection_matchers", "~> 1.1.2"
+  gem "rspec-json_expectations", "~> 2.1"
```

Comment:
Can you fix the indention here (pronto doesn't see it, because rubocop complains about the line below).